cluster/vagrant/provision-master.sh

```diff
@@ -108,8 +104,8 @@ fi
 if ! which /usr/libexec/cockpit-ws &>/dev/null; then
 
   pushd /etc/yum.repos.d
-    wget https://copr.fedoraproject.org/coprs/sgallagh/cockpit-preview/repo/fedora-21/sgallagh-cockpit-preview-fedora-21.repo
-    yum install -y cockpit cockpit-kubernetes
+    wget https://copr.fedoraproject.org/coprs/sgallagh/cockpit-preview/repo/fedora-22/sgallagh-cockpit-preview-fedora-22.repo
```

Comment:
@marun Forgot to reply sorry. No, wget doesn't exit under fedora. See above^

>I've test with libvirt but I had to add this dnf install -y wget because the libvirt fedora 23 box doesn't >have wget installed. However I got some errors but I don't know if they are related to libvirt fedora >box or kube: